### PR TITLE
Style ResourceInfo to be aside resource images

### LINKED ISSUE #548: Add type filters to "Omat tilat" admin view
Note, only type filters need to be added in this issue.

- [x] The type filters should function as toggles. Show a resource if its type is among the selected types. Filtering should be done in the frontend.
- [x] By default all types should be selected.
- [x] Only show types that at least one of the resources in the view has. The types can be get from resource data.
- [x] The types are not translated so only the Finnish name can be shown.

omat-tilat-filters

### PR DESCRIPTION
**Pull request by YoYuUm**
Closes #548

### CODE REVIEW

File: app/pages/resource/resource-info/resource-info.less [hunk 1]
```diff
@@ -17,7 +17,25 @@
...
+  .details-row {
+    font-weight: 600;
+  }
+
+  .resource-equipment {
+    .label {
+      display: inline-block;
+      font-size: @font-size-small;
+      padding: 0.4em;
+      margin: 0.2em;
```
Comment (lraivio) [inline app/pages/resource/resource-info/resource-info.less]: - [ ] Probably better to not have left margin in the first element.

**Comment by YoYuUm** [comment]
Fixed

**Comment by lraivio** [comment]
- [ ] You can squash the fixup 👍

**Comment by lraivio** [comment]
- [ ] Just noticed that the favorite button is not vertically aligned with the other content. You can fix that straight without a separate fixup commit.

**Pull request merged**